### ISSUE TITLE
Notification or information about new publications

### ISSUE THREAD

**Issue by siggi1984** [fast4x/RiMusic#20]
### Description

Would it be possible to get a notification or information about new releases of singles and albums of the artists you have marked to stay up to date.

### Suggested Solution

Would such a function be possible?

**Comment by siggi1984** [comment]
Maybe it can be arranged that the new or last published singles or albums of the marked artists are displayed under the quick picks.

![Screenshot_20231014-214826](https://github.com/fast4x/RiMusic/assets/103368536/5c70ebdf-fd27-42b6-8516-cda555f2bc37)

**Comment by fast4x** [comment]
Added partially, if possibile will be available on version 0.6.9

**Comment by fast4x** [comment]
Implemented, available from version 0.6.9

To not weigh down quickpics, i created a new page with new releases, includes moods and genres

**Comment by siggi1984** [comment]
> Implemented, available from version 0.6.9
> 
> To not weigh down quickpics, i created a new page with new releases, includes moods and genres

Cool, I'm looking forward to the function

**Comment by siggi1984** [comment]
Hi, the new section is really cool.

But with the new releases I actually meant that only the new releases of the marked artists in my own library are displayed.

New releases are displayed but from artists I don't know and don't listen to, I want to follow the new releases of my favorite artists.

I don't know if and to what extent something like this would be feasible?

**Comment by fast4x** [comment]
Yes, you’re right, I’m testing how to best develop it, the starting point is to have the news

**Comment by fast4x** [comment]
Do you want reopen this?

**Comment by siggi1984** [comment]
> Do you want reopen this?

Yes, please.

**Comment by fast4x** [comment]
Reopen for implementing preferred artist from personal list

**Comment by fast4x** [comment]
Implemented, Added new albums from preferites artists on discovery and quick pics pages.

Available from version 0.6.11
